### PR TITLE
Fix minor issues

### LINKED ISSUE #129: Problem 123: hint repeated in main text
Hi,
in problem 123 the hint is repeated in the main text verbatim. I'm assuming this was not intentional.

### PR DESCRIPTION
**Pull request by oscarlevin**
This will close #129 and #136, as well as add math mode around some stray variables in hints.  Will now also address the typo in #139

### CODE REVIEW

**Comment by oscarlevin** [comment]
I'm not sure why this has so many extra commits.  Probably an issue with how we have been merging and updating our forks.

**Comment by sphaso** [comment]
@oscarlevin those extra commits should go away once you rebase your branch `oscarlevin:typos` from `OpenDiscreteMath:update-bogart`

File: mbx/s2-2-recurrence.mbx [hunk 1]
```diff
@@ -17,7 +17,7 @@
...
+              Suppose that our <m>n</m>-element set is <m>N = \{a_1,a_2,\dots ,a_n\}</m>. Then a subset of <m>N</m> either contains an or it doesn't. In our discussion of the Pascal recurrence, we showed that the number of <m>k</m>-element subsets of <m>N</m> that contain an is the same as the number of <m>(k - 1)</m>-element subsets of <m>N - \{a_n\}</m>. The bijection we used to prove this consists of taking an away from a set containing <m>a_n</m>. Thus the number of subsets of <m>N</m> containing <m>a_n</m> is the same as the number of subsets of the <m>(n - 1)</m>-element set <m>N - \{a_n\}</m>. But the subsets of <m>N</m> not containing an are exactly the same as the subsets of <m>N - \{a_n\}</m>. Thus we can partition the subsets of <m>N</m> into two blocks, each of which has size equal to the number of subsets of <m>N - \{a_n\}</m>. Therefore, by the sum principle, the number of subsets of <m>N</m> is twice the number of subsets of <m>N - \{a_n\}</m>.
```
Comment (mitchkeller) [inline mbx/s2-2-recurrence.mbx]: If you're fixing this paragraph, there are (at least) three occurrences of `an` that should be `<m>a_n</m>`.

**Comment by mitchkeller** [comment]
Don't rebase now that you've pushed your branch to Github, but @sphaso is correct that generally the right way to avoid all the messy commit history is to do a rebase right before pushing. If you fix the issues I noted in my one comment, then I think this should be good to merge.

**Comment by oscarlevin** [comment]
I've made the suggested changes.  Should be good now.

**Pull request merged**